docs/installation.rst

```diff
@@ -0,0 +1,69 @@
+
+#. Install Anaconda's Miniconda following the steps at https://conda.io/docs/user-guide/install/, making sure to select the Python-3 version. If you're on Linux /macOS, it also assumes that you installed Miniconda-3 to your home directory.
+
+#. Start a terminal (Linux / macOS) or the Anaconda prompt (Windows). Activate the root (or base) environment of your Miniconda and create an environment containing Parcels, all its essential dependencies, and the nice-to-have cartopy and jupyter packages:
+
+    .. code-block:: bash
+
+      conda activate root  # Linux / macOS
+      activate root        # Windows
+
+      conda create -n parcels -c conda-forge parcels cartopy
+
+    .. note::
+
+        For some of the examples, ``pytest`` needs to be installed with ``conda install -n parcels pytest``
+
+#. Activate the newly created Parcels environment:
+
+    .. code-block:: bash
+
+      conda activate parcels  # Linux / macOS
+      activate parcels        # Windows</code></pre>
+
+#. Get a zipped copy of the Parcels tutorials and examples from `here <https://docs.oceanparcels.org/en/latest/_downloads/307c382eb1813dc691e8a80d6c0098f7/parcels_tutorials.zip>`_ and unzip it.
+
+#. Go to the unzipped folder and run one of the examples to validate that you have a working Parcels setup:
+
+    .. code-block:: bash
+
+      python example_peninsula.py --fieldset 100 100
+
+    .. note::
+      If you are on macOS and get a compilation error, you may need to accept the Apple xcode license ``xcode-select --install``. If this does not solve the compilation error, you may want to try running ``export CC=gcc``. If the compilation error remains, you may want to check `this solution <https://stackoverflow.com/a/58323411/5172570>`_.
+
+#. Optionally, if you want to run all the examples and tutorials, start Jupyter and open the tutorial notebooks:
+
+    .. code-block:: bash
+
+      jupyter notebook
+
+#. The next time you start a terminal and want to work with Parcels, activate the environment with:
+
+    .. code-block:: bash
+
+      conda activate parcels  # Linux / macOS
+      activate parcels        # Windows
+
+
+Installation for developers
+===========================
+
+Parcels depends on a working Python installation, a netCDF installation, a C compiler, and various Python packages. If you prefer to maintain your own Python installation providing all this, download one of the environment files for `linux <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment_py3_linux.yml>`_, for `macOS <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment_py3_osx.yml>`_ or for `Windows <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment_py3_win.yml>`_ and install it with:
+
+    .. code-block:: bash
+
+      git clone https://github.com/OceanParcels/parcels.git
+      conda env create -f environment_py3_<OS>.yml  # where <OS> is either linux, osx or win
```

Comment:
I'm not sure about the wording of this section. Is it necessary to include info on downloading the environment files? (they would be in the clone anyway). Perhaps something along the line of 

```suggestion
If you would prefer to have a development installation of Parcels (i.e., where the code can be actively editted), you can do so by cloning the Parcels repo, installing dependencies using the environment file (which includes Python, netCDF tooling, a C compiler, and various Python packages), and then installing Parcels in an editable mode such that changes to the cloned code can be tested during development.

    .. code-block:: bash

      git clone https://github.com/OceanParcels/parcels.git
      conda env create -f environment_py3_<OS>.yml  # where <OS> is either linux, osx or win
```